app.rb

```diff
@@ -142,13 +145,26 @@ def process(choice)
     :continue
   end
 
+  def save_data
+    File.write('books.json', @books.to_json)
+    File.write('people.json', @people.to_json)
+    File.write('rentals.json', @rentals.to_json)
+  end
+
+  def load_data
+    @books = JSON.parse(File.read('books.json')) if File.exist?('books.json')
+    @people = JSON.parse(File.read('people.json')) if File.exist?('people.json')
+    @rentals = JSON.parse(File.read('rentals.json')) if File.exist?('rentals.json')
+  end
```

Comment:
Nice job so far👍🏼. It is a requirement in this milestone that your data should be saved in sush a way that when you restart the app, the data is retained😉.
<img width="1066" alt="image" src="https://github.com/Gardimy/OOP-school-library/assets/64643995/4733be0b-cf47-4261-8a95-2307f359f5a6">

But currently, the app breaks when you try to retrieve the data. also, the option to exit your app (7) doesn't work properly, you can do well to fix that as it gives a bad user experience.
<img width="1207" alt="image" src="https://github.com/Gardimy/OOP-school-library/assets/64643995/b17c5586-de6c-4282-aaeb-df4c8f3bc900">


Kindly do well to fix these issues😉.